pkg/apic/subscription.go

```diff
@@ -36,6 +36,7 @@ type Subscription interface {
 	GetID() string
 	GetName() string
 	GetApicID() string
+	GetRemoteAPIAttributes() map[string]string
```

Comment:
Adding all the attributes from the API Service to the subscription because the mulesoft agent does not rely on the remoteAPIID attribute to determine anything. It needs other attributes to deal with the specific revision.